### PR TITLE
Category

### PR DESCRIPTION
**Pull request by christo-p**

### CODE REVIEW

File: quizz/src/App.css [hunk 1]
```diff
@@ -1,26 +1,3 @@
-
-
-
 .App {
```
Comment (ghost) [inline quizz/src/App.css:1]: If App.css is not used at all, just delete it

File: quizz/src/components/CardQuestion.js [hunk 2]
```diff
@@ -45,6 +45,8 @@ class CardQuestion extends React.Component {
     }  
    
     render() {
+        console.log(this.state.arrayQuestions)
```
Comment (ghost) [inline quizz/src/components/CardQuestion.js]: console.log are not kept in dev

**Review by ghost — CHANGES_REQUESTED** [review]
console.log & App.css to delete ?

File: quizz/src/components/CardQuestion.js (hunk 2 above)
Comment (christo-p) [reply quizz/src/components/CardQuestion.js]: done

**Review by ghost — APPROVED** [review]

**Pull request merged**